apps/feedback/models/schedule.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"time"
+	"github.com/jinzhu/gorm"
+
+	userModels "github.com/iReflect/reflect-app/apps/user/models"
+)
+
+type Schedule struct {
+	gorm.Model
+	Team         userModels.Team `gorm:"ForeignKey:TeamID; AssociationForeignKey:ID"`
+	TeamID       uint            `gorm:"not null"`
+	PeriodValue  uint            `gorm:"not null"`
```

Comment:
Not clear what is the use of all the period fields. We should add documentation throughout the code to explain.

Reply:
done